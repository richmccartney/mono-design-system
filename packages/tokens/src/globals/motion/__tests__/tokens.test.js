const motionTokens = require('../tokens.json');

describe('motion tokens', () => {
  test(': matches snapshot', () => {
    expect(motionTokens).toMatchSnapshot();
  });
});
