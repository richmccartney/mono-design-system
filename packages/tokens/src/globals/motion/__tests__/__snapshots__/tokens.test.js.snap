// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`motion tokens : matches snapshot 1`] = `
Object {
  "motion": Object {
    "duration": Object {
      "fast": Object {
        "value": ".15s",
      },
      "medium": Object {
        "value": ".3s",
      },
      "slow": Object {
        "value": ".6s",
      },
    },
    "ease": Object {
      "in": Object {
        "value": "cubic-bezier(.4, 0, .7, .2)",
      },
      "inout": Object {
        "value": "cubic-bezier(.4, 0, .2, 1)",
      },
      "out": Object {
        "value": "cubic-bezier(0, 0, .2, 1)",
      },
    },
  },
}
`;
